tests/cmds/test_users.py

```diff
@@ -169,6 +226,32 @@ def test_list_users_when_given_excluding_active_and_inactive_uses_active_equals_
     )
 
 
+def test_add_legal_hold_membership_to_user_dataframe_adds_legal_hold_columns_to_dataframe(
+    cli_state, get_all_matter_success, get_all_custodian_success
+):
+    testdf = DataFrame.from_records(
+        [{"userUid": "840103986007089121", "status": "Active"}]
+    )
+    result = _add_legal_hold_membership_to_user_dataframe(cli_state.sdk, testdf)
+    assert "legalHoldUid" in result.columns
+    assert "legalHoldName" in result.columns
+
+
+def test_list_include_legal_hold_membership_merges_in_and_concats_legal_hold_info(
+    runner,
+    cli_state,
+    get_all_users_success,
+    get_all_custodian_success,
+    get_all_matter_success,
+):
+    result = runner.invoke(
+        cli, ["users", "list", "--include-legal-hold-membership"], obj=cli_state
+    )
+
+    assert "Legal Hold #1,Legal Hold #2" in result.output
+    assert "123456789,987654321" in result.output
+
+
```

Comment:
Can we get a test for the following two cases:

1.) A user is not on any legal hold but the `--include-legal-hold-membership` flag was passed in (keys should be there but their values should be `None`)
2.) A user is on a legal hold but the flag `--include-legal-hold-membership` was not passed in (keys and values should not be present in the output)

Reply:
I added these tests. The code didn't handle cases where 0 custodians were returned. Thanks for suggesting this test case.

I also added a test for cases where the flag`--include-legal-hold-membership` and no matters were returned.